pulsar-client-admin-api/pom.xml

```diff
@@ -33,16 +33,16 @@
     <name>Pulsar Client Admin :: API</name>
 
     <dependencies>
+
         <dependency>
             <groupId>${project.groupId}</groupId>
             <artifactId>pulsar-common</artifactId>
             <version>${project.parent.version}</version>
         </dependency>
 
         <dependency>
-            <groupId>${project.groupId}</groupId>
-            <artifactId>pulsar-client-original</artifactId>
-            <version>${project.parent.version}</version>
+            <groupId>com.google.code.gson</groupId>
+            <artifactId>gson</artifactId>
```

Comment:
if gson is not already a dependency imported by  pulsar-client-original in 2.7.x I would like to not add it.
Users of pulsar-client-original already have Jackson and usually who uses pulsar-client-admin-api uses pulsar-client, so I would lean toward using only Jackson.

what's your plan for removing gson in a follow up patch ? this next patch will become a blocker for the 2.8 release

Reply:
Adding `gson` as the dependency is a temporary action, with removal of `pulsar-client-original` and `pulsar-package-core`, `pulsar-client-admin-api` will lose dependency of `gson`, but we still have a lot of classes import `gson`, such as https://github.com/apache/pulsar/blob/581fd5b01e6775db1429f645ffe2ce7d984c2877/pulsar-client-admin-api/src/main/java/org/apache/pulsar/client/admin/Topics.java#L21. 

I do understand that the 2.8 release is in a hurry, but there are indeed many problems that need to be fixed one by one in order to achieve the ultimate goal. So the removal of `gson` will be in a separate PR soon.

Reply:
My problem was about "adding" gson, if it was not already there, so to me this is not a problem.

thanks for your clarification